### ISSUE TITLE
GUI templates does not respect layouts.

### ISSUE THREAD

**Issue by mosmetro** [defold/editor2-issues#1124]
GUI templates does not respect layouts.

<table>
<tr><td>Defold version</td><td>1.2.111</td></tr>
<tr><td>Defold sha</td><td>8975f56e27df2fa15395dbd44aea897bdad29064</td></tr>
<tr><td>Build time</td><td>2017-08-14T10:14:50.341779</td></tr>
<tr><td>OS name</td><td>Mac OS X</td></tr>
<tr><td>OS version</td><td>10.12.6</td></tr>
<tr><td>OS arch</td><td>x86_64</td></tr>
<tr><td>Java version</td><td>1.8.0_102-b14</td></tr>
<tr><td>GPU</td><td>Intel HD Graphics 4000 OpenGL Engine</td></tr>
<tr><td>GPU Driver</td><td>2.1 INTEL-10.25.17</td></tr>
<table>

**Comment by erikangelin-king** [comment]
https://forum.defold.com/t/in-editor-2-gui-templates-still-does-not-respect-layouts/13818

**Comment by mosmetro** [comment]
Does this will be ever fixed? This bug prevents the use of GUI templates in editor 2.

**Comment by mosmetro** [comment]
It's a shame that this not fixed yet.

**Comment by erikangelin-king** [comment]
I agree, and we picked it up about a month ago but making it work the way we want has turned out to be a lot trickier than we expected.

**Comment by AGulev** [comment]
Related, I think https://github.com/defold/editor2-issues/issues/2044

**Comment by IVA-apps** [comment]
Solve this problem, please! =)

**Comment by lrxit** [comment]
Very annoying problem!!!

**Comment by rezam90** [comment]
any updates?

**Comment by eangelin** [comment]
Sadly no updates :( We've thrown around a few ideas on how to solve it but nothing really "fits" into the architecture we use in the editor. Either they are too clumpsy/flaky, too resource consuming or don't really solve the entire problem. We've even semi-seriously discussed removing layouts alltogether :) It's one of these annoying problems that stick in the back of your head. There is probably a really simple solution that will make us feel like idiots, we just haven't found it yet.
